src/Integration.Vsix/CFamily/ProcessRunnerArguments.cs

```diff
@@ -71,15 +67,16 @@ public ProcessRunnerArguments(string exeName, bool isBatchScript)
 
         public string WorkingDirectory { get; set; }
 
-        public int TimeoutInMilliseconds { get; set; }
-
         private bool IsBatchScript { get; set; }
 
         /// <summary>
         /// Additional environments variables that should be set/overridden for the process. Can be null.
         /// </summary>
         public IDictionary<string, string> EnvironmentVariables { get; set; }
 
+        public Action<StreamWriter> HandleInputStream { get; set; } = writer => { };
```

Comment:
@duncanp-sonar assigning to no-op so the actions won't be null.

Reply:
A null coalesce at the point of invocation would make it clearer that nulls are ok; validating the arguments and throwing on null would make it clear that nulls are not ok. Setting a non-null default but allowing it to be changed to null isn't as clear.